app/swagger/schemas/gibct/institution_programs.rb

```diff
@@ -37,20 +37,34 @@ class InstitutionPrograms
-                property :facility_code, type: :string, example: '1X12345'
-                property :institution_name, type: :string, example: 'CODE PLACE'
-                property :city, type: %i[null string], example: 'ANYTOWN'
-                property :state, type: %i[null string], example: 'WA'
-                property :country, type: %i[null string], example: 'USA'
-                property :preferred_provider, type: %i[null boolean], example: false
-                property :tuition_amount, type: %i[null integer], example: 1000
-                property :va_bah, type: %i[null number], example: 220
-                property :dod_bah, type: %i[null integer], example: 200
+                property :program_type, type: :string, example: 'NCD',
+                                        description: 'The classification of the program, ex: IHL, NCD, OJT'
+                property :description, type: :string, example: 'Computer Science',
+                                       description: 'Program name'
+                property :length_in_hours, type: :string, example: '680',
+                                           description: 'Length of program (in hours)'
+                property :length_in_weeks, type: :integer, example: 12,
+                                           description: 'Length of program (in weeks)'
+                property :facility_code, type: :string, example: '1X12345',
+                                         description: 'Program institution facility code'
+                property :institution_name, type: :string, example: 'CODE PLACE',
+                                            description: 'Program institution name'
+                property :city, type: :string, example: 'ANYTOWN',
+                                description: 'Program institution physical city location'
+                property :state, type: :string, example: 'WA',
+                                 description: 'Program institution physical state location'
+                property :country, type: :string, example: 'USA',
+                                   description: 'Program institution physical country location'
+                property :preferred_provider, type: :boolean, example: FALSE,
+                                              description: 'Program institution preferred provider indicator;
+                                                            a provider that takes on the costs of the veterans
+                                                            education if the requirements are not met'
+                property :tuition_amount, type: :integer, example: 1000,
+                                          description: 'Program tuition amount'
+                property :va_bah, type: :number, example: 220,
+                                  description: 'VA Basic Allowance for Housing'
+                property :dod_bah, type: :integer, example: 200,
```

Comment:
Should both of these be `number` (or `integer`)?

Reply:
VA_bah is a float and dod_bah is an int in the database.